MongoConsumerLibary/MongoConnection/Collections/BaseBoxCollection.cs

```diff
@@ -0,0 +1,11 @@
+﻿using MongoConsumerLibary.MongoConnection.Enums;
+using System.Collections.Generic;
+
+namespace MongoConsumerLibary.MongoConnection.Collections
+{
+    public class BaseBoxCollection : ExpirableCollection
+    {
+        public BaseBoxType IcdType { get; set; }
+        public Dictionary<string,(int,bool)> DecryptedParameters { get; set; }
```

Comment:
why bool?

Reply:
The dictionary coming from Kafka includes a bool for indicating if the parameter is legal or not

Reply:
Ok, put only the legal in the mongoDB and the type of the variable will be Dictionary<string, int>